fetch.bs

```diff
@@ -3989,11 +3993,20 @@ steps:
    <li>Matching <var>request</var>'s <a for=request>current URL</a>'s <a for=url>host</a> per
    <a href=https://datatracker.ietf.org/doc/html/rfc6797#section-8.2>Known HSTS Host Domain Name Matching</a>
    results in either a superdomain match with an asserted <code>includeSubDomains</code> directive
-   or a congruent match (with or without an asserted <code>includeSubDomains</code> directive).
-   [[!HSTS]]
+   or a congruent match (with or without an asserted <code>includeSubDomains</code> directive); or
+   DNS resolution for the request finds a matching HTTPS RR per
+   <a href=https://datatracker.ietf.org/doc/html/draft-ietf-dnsop-svcb-https#section-8.5>HTTP Strict Transport Security</a>.
```

Comment:
The link "HTTP Strict Transport Security" should be to section 9.5, but it's also a little confusing because "HTTP Strict Transport Security" is the name of the other feature. Perhaps...

```suggestion
   or a congruent match (with or without an asserted <code>includeSubDomains</code> directive) [[!HSTS]]; or
   DNS resolution for the request finds a matching HTTPS RR per
   <a href=https://datatracker.ietf.org/doc/html/draft-ietf-dnsop-svcb-https#section-9.5>section 9.5</a>
   of [[!SVCB]].
```